wrong docs in CONTRIBUTING.md

The docs tell what env-vars to set, but some apparently need to be set to a specific value:

```
-- FAIL: TestAccComputeRegionBackendService_withBackendAndUpdate (0.00s)
    provider_test.go:91: One of GOOGLE_REGION, GCLOUD_REGION, CLOUDSDK_COMPUTE_REGION must be set to us-central1 for acceptance tests
--- FAIL: TestAccComputeTargetTcpProxy_TargetTcpProxyBasicExample (0.01s)
    provider_test.go:91: One of GOOGLE_REGION, GCLOUD_REGION, CLOUDSDK_COMPUTE_REGION must be set to us-central1 for acceptance tests
```

here `GOOGLE_REGION=us-central1` is required.

Other variables like GOOGLE_ORG are not obvious to what values they should be set.

Finally I'd expect to see what command I should use to run all tests (just `make testacc`?)

--- a/.github/CONTRIBUTING.md
+++ b/.github/CONTRIBUTING.md
@@ -43,12 +43,16 @@ GOOGLE_ORG
 GOOGLE_BILLING_ACCOUNT
 ```
 
+The only region we support running tests in right now is `us-central1` - some products that are tested here are only available in a few regions, and the only region that all products are available in is `us-central1`.
+
 To run a specific test, use a command such as:
 ```
 make testacc TEST=./google TESTARGS='-run=TestAccContainerNodePool_basic'
 ```
 
 The `TESTARGS` variable is regexp-like, so multiple tests can be run in parallel by specifying a common substring of those tests (for example, `TestAccContainerNodePool` to run all node pool tests).
+
+To run all tests, you can simply use `TESTARGS='-run=TestAcc'` - but please keep in mind that that is quite a few tests and will take quite a long time and create some fairly expensive resources.  It usually is not advisable to run all tests.
 
 ### Writing Tests
 

--- a/.github/CONTRIBUTING.md
+++ b/.github/CONTRIBUTING.md
@@ -52,7 +52,7 @@ make testacc TEST=./google TESTARGS='-run=TestAccContainerNodePool_basic'
 
 The `TESTARGS` variable is regexp-like, so multiple tests can be run in parallel by specifying a common substring of those tests (for example, `TestAccContainerNodePool` to run all node pool tests).
 
-To run all tests, you can simply use `TESTARGS='-run=TestAcc'` - but please keep in mind that that is quite a few tests and will take quite a long time and create some fairly expensive resources.  It usually is not advisable to run all tests.
+To run all tests, you can simply omit the `TESTARGS` argument - but please keep in mind that that is quite a few tests and will take quite a long time and create some fairly expensive resources.  It usually is not advisable to run all tests.
 
 ### Writing Tests
 

